app/static/js/load_weeks.js

```diff
@@ -0,0 +1,43 @@
+document.onload = function () {
```

Comment:
All the rest should also be called from here, to make sure `container` is defined.

Reply:
I entered all of the initial settings to a 'DOMContentLoaded' with the relevant settings functions. 
(Its been added as part of the solution we discussed about: "repetitions in code")